lightning/src/ln/channelmanager.rs

```diff
@@ -2537,7 +2561,7 @@ impl<Signer: Sign, M: Deref, T: Deref, K: Deref, F: Deref, L: Deref> ChannelMana
 	/// May be called with channel_state already locked!
 	fn get_channel_update_for_unicast(&self, chan: &Channel<Signer>) -> Result<msgs::ChannelUpdate, LightningError> {
 		log_trace!(self.logger, "Attempting to generate channel update for channel {}", log_bytes!(chan.channel_id()));
-		let short_channel_id = match chan.get_short_channel_id() {
+		let short_channel_id = match chan.get_short_channel_id().or(chan.latest_inbound_scid_alias()) {
```

Comment:
May need docs updates for `latest_inbound_scid_alias` in `channel.rs` for the additional `channel_update` usages